doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -0,0 +1,255 @@
+AP Pipeline Tutorial
+####################
+
+Setup
+=====
+
+Pick up where you left off in :doc:`Getting Started <getting-started>`.
+This means you already have a repository of ingested DECam data and have setup
+the LSST Science Pipelines stack as well as ``ap_pipe`` and ``ap_association``.
+
+Your directory structure should look something like 
+
+.. code-block:: none
+
+   repo/   <-- where you will be running ap_pipe
+      ingested/
+         registry.sqlite3   <--- created during image ingestion
+         image1.fits   <--- linked here by default during image ingestion
+         image2.fits
+         ...
+         ref_cats/   <--- copied or linked here manually by you
+            gaia/
+               shard1
+               shard2
+               ...
+            pan-starrs/
+               shard1
+               shard2
+               ...
+      calibs/
+         calibRegistry.sqlite3   <--- created/updated during calib/defect ingestion
+         cpBIAS/
+            bias1.fits   <--- linked here by default during calib ingestion
+            bias2.fits
+            ...
+         cpFLAT/
+            flat1.fits   <--- linked here by default during calib ingestion
+            flat2.fits
+            ...
+         defects/   <--- put here manually by you prior to defect ingestion
+            defect1.fits
+            defect2.fits
+            ...
+      templates/   <--- copied or linked here manually by you
+         repositoryCfg.yaml
+         deepCoadd/
+            g/
+               0/
+                  psfMatched-0,0.fits
+                  psfMatched-0,1.fits
+                  ...
+
+.. _section-ap-pipe-command-line:
+
+AP Pipeline on the command line
+===============================
+
+The executable to run for the AP Pipeline (`lsst.ap.pipe.ApPipeTask`) is
+in ``ap_pipe/bin/ap_pipe.py``.
+
+To process your ingested data, run
+
+.. prompt:: bash
+   
+   ap_pipe.py repo --calib repo/calibs --rerun processed --id visit=123456 ccdnum=42 filter=g
+
+In this case, a ``processed`` directory will be created within
+``repo/rerun`` and the results will be written there.
+Using coadds as templates makes `reruns 
+<https://pipelines.lsst.io/getting-started/processccd.html#aside-reruns-and-output-butler-repositories>`_
+useful for many workflows,
+because it is possible to run ``ap_pipe`` many times while using the same
+repository of coadd templates as both an input and output location on disk.
+
+This example command only processes observations that have a ``dataId``
+corresponding to visit 123456 and ccdnum 42 in with a filter called g.
+You must specify a ``dataId`` string or no data will be processed.
+`lsst.ap.pipe` supports ``dataId`` parsing, e.g., ``ccdnum=3^6..12`` will process
+``ccdnums`` 3, 6, 7, 8, 9, 10, 11, and 12.
+
+.. note::
+
+   Until a resolution for `DM-12672 <https://jira.lsstcorp.org/browse/DM-12672>`_
+   is found, you should include a filter in the ``dataId`` string for
+   ``ap_pipe`` to run successfully.
```

Comment:
I don't think it's a good idea to link to Jira issues from user-facing documentation, especially in cases like this where a user who doesn't have Jira access will have no way of figuring out what's going on. Why not simply say that some instruments may require you to explicitly give a filter, that this is a known issue, but that you don't know when it will be fixed?